### ISSUE TITLE
Including non-reactive widgets within Preact elements

### ISSUE THREAD

**Issue by whamtet** [developit/preact#562]
I often wish to add non-reactive widgets inside a Preact element.  I do this by adding the widget during componentDidMount

```
...
componentDidMount() {
//add autoComplete widget
$('#myPreactComponent').autoComplete({ ...
```

However this creates problems with the virtual dom, e.g. the widget pops up in another random place.  Is there a safe way to exclude the non-reactive widget from Preact.  Alternately I could use css to place the widget outside of the Preact element but position it within.  Any advice much appreciated.

**Comment by developit** [comment]
@whamtet For now, I would recommend removing the non-react widget from `componentWillUnmount()`:

```js
componentWillUnmount() {
  // just grab a reference to the externally created element and remove it:
  this.base.querySelector('.autocomplete').remove();
}
```

This will be fixed in 8.0.

**Comment by whamtet** [comment]
Thanks Jason!

**Comment by developit** [comment]
This should be fixed in 8.0!
